man/org.freedesktop.oom1.xml

```diff
@@ -56,12 +59,37 @@ node /org/freedesktop/oom1 {
 
     <variablelist class="dbus-method" generated="True" extra-ref="DumpByFileDescriptor()"/>
 
+    <variablelist class="dbus-signal" generated="True" extra-ref="Killed"/>
+
     <!--End of Autogenerated section-->
 
     <refsect2>
       <title>Methods</title>
 
-      <para>...</para>
+      <para><function>Killed</function> signal is sent when any cgroup is killed by oomd.</para>
+      <table>
+        <title>Killing reasons</title>
+        <tgroup cols="2" align="left" colsep="1" rowsep="1">
+        <colspec colname="reason"/>
+        <colspec colname="description"/>
+        <thead>
+          <row>
+            <entry>Reason</entry>
+            <entry>Description</entry>
+          </row>
+        </thead>
+        <tbody>
+          <row>
+            <entry>memory-used</entry>
+            <entry>application took too much memory</entry>
```

Comment:
nit: memory and swap.

Swap policy doesn't kill solely based on memory

Reply:
Also this should explicitly say that more reasons might added in the future

Reply:
> Also this should explicitly say that more reasons might added in the future

any examples I should follow?

Reply:
Something along those lines will suffice, it's just to ensure users now the list can and will be expanded in the future:

```
<para><function>Killed</function> signal is sent when any cgroup is killed by oomd.
Note that more reasons will be added in the future, and the table below will be expanded accordingly.</para>
```